Documentation/devel/pod-lifecycle.md

```diff
@@ -51,7 +55,9 @@ After preparation completes, while still holding the exclusive lock (the lock is
 
 `rkt run` transitions directly from #Prepare to #Run by renaming `$var/prepare/$uuid` to `$var/run/$uuid`, entirely skipping the #Prepared phase.
```

Comment:
The first sentence in this document explains it - it basically to avoid repeating longer paths:

Throughout this document `$var` is used to refer to the directory `/var/lib/rkt/pods`, and `$uuid` refers to a pod's UUID e.g. "076292e6-54c4-4cc8-9fa7-679c5f7dcfd3".